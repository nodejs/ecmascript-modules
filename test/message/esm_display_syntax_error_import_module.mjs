// Flags:  --experimental-modules
/* eslint-disable node-core/required-modules */
import '../common/index.mjs';
import '../fixtures/es-module-loaders/syntax-error-import.mjs';
