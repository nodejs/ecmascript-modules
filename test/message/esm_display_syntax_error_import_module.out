(node:*) ExperimentalWarning: The ESM module loader is experimental.
file:///*/test/fixtures/es-module-loaders/syntax-error-import.mjs:1
import { foo, notfound } from './module-named-exports.mjs';
              ^^^^^^^^
SyntaxError: The requested module './module-named-exports.mjs' does not provide an export named 'notfound'
    at ModuleJob._instantiate (internal/modules/esm/module_job.js:*:*)
